tests/test_data.py

```diff
@@ -1,25 +1,25 @@
+test_admin_user = {
```

Comment:
I'm curious, what's the reason to change this test data from `dict()` to a `{` `}`? Is it more efficient like that?

Reply:
@isabelcosta  Yes it is faster and more efficient. Please go through this. for more info https://medium.com/@jodylecompte/dict-vs-in-python-whats-the-big-deal-anyway-73e251df8398